bpf/process/bpf_process_event.h

```diff
@@ -202,8 +202,7 @@ FUNC_INLINE long cwd_read(struct cwd_read_data *data)
 			probe_read(&data->dentry, sizeof(data->dentry),
 				   _(&mnt->mnt_mountpoint));
 			data->mnt = parent;
-			probe_read(&data->vfsmnt, sizeof(data->vfsmnt),
-				   _(&mnt->mnt));
+			data->vfsmnt = &parent->mnt;
```

Comment:
ok,  parent->mnt is not pointer but actual vfsmnt object embedded in mount struct,
so we can just take the address from parent.. but I think you still need to use _() to get CORE involved

Reply:
OK， i will fix it